tests/uniffi_logger.rs

```diff
@@ -0,0 +1,42 @@
+use telio;
+use telio::UniffiTelioTracingSubscriber;
+
+mod test_module {
+    use std::sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc,
+    };
+
+    use telio::ffi_types::TelioLoggerCb;
+
+    use super::*;
+
+    #[test]
+    fn uniffi_test_logger() {
```

Comment:
`tracing::set_global_default` is necessary to make this test work, but there can't be called twice in the same test file, that's why this test got it's own file instead of also placing it in `logger.rs`. The duplication will be fixed in a later PR